app/components/bali/calendar/header/component.html.erb

```diff
@@ -0,0 +1,32 @@
+<div class="header mb-4">
+  <div class="columns is-mobile is-gapless">
+    <div class="coulum is-narrow">
```

Comment:
```suggestion
    <div class="column is-narrow">
```

Reply:
Just verified if this still need it, 'cause is a typo and still workin'.. Maybe is not necesary or there is some other issue on the html.